docs/reference/supervisor.md

```diff
@@ -235,6 +241,8 @@ node = wb_supervisor_node_get_parent_node(node)
 | `/supervisor/node/get_id` | `service` | `webots_ros::node_get_id` | `uint64 node`<br/>`---`<br/>`int32 id` |
 | `/supervisor/node/get_def` | `service` | `webots_ros::node_get_name` | `uint64 node`<br/>`---`<br/>`string name` |
 | `/supervisor/node/get_parent_node` | `service` | `webots_ros::node_get_parent_node` | `uint64 node`<br/>`---`<br/>`uint64 node` |
+| `/supervisor/node/is_proto` | `service` | `webots_ros::node_is_proto` | `uint64 node`<br/>`---`<br/>`bool value` |
+| `/supervisor/get_from_def` | `service` | `webots_ros::supervisor_get_from_def` | `string name`<br/>`uint64 proto`<br/>`---`<br/>`uint64 node` |
```

Comment:
If we want to use the same service both for the wb_supervisor_node_get_from_def and wb_supervisor_node_get_from_proto_def (as it is currently), then yes the order is correct and the `proto` parameter is the last because is optional.

Reply:
OK, Thank you for the explanation.